tests/integration/test_schema.py

```diff
@@ -2316,21 +2322,26 @@ async def test_invalid_schema_should_provide_good_error_messages(registry_async_
     )
     assert res.json()["message"] == "Invalid AVRO schema. Error: Expecting ',' delimiter: line 1 column 18 (char 17)"
 
-    # Unfortunately the AVRO library doesn't provide a good error message, it just raises an TypeError
     schema_str = jsonlib.dumps({"type": "enum", "name": "error"})
     res = await registry_async_client.post(
         f"subjects/{test_subject}/versions",
         json={"schema": schema_str},
     )
-    assert res.json()["message"] == "Invalid AVRO schema. Error: Provided schema is not valid"
+    assert (
+        res.json()["message"]
+        == "Invalid AVRO schema. Error: Enum symbols must be a sequence of strings, but it is <class 'NoneType'>"
+    )
 
     # This is an upstream bug in the python AVRO library, until the bug is fixed we should at least have a nice error message
     schema_str = jsonlib.dumps({"type": "enum", "name": "error", "symbols": {}})
     res = await registry_async_client.post(
         f"subjects/{test_subject}/versions",
         json={"schema": schema_str},
     )
-    assert res.json()["message"] == "Invalid AVRO schema. Error: error is a reserved type name."
+    assert (
+        res.json()["message"]
+        == "Invalid AVRO schema. Error: Enum symbols must be a sequence of strings, but it is <class 'dict'>"
```

Comment:
Same here. This is not understandable error message if received by Schema Registry users